### PR TITLE
Display number of likes for each items on the homepage, displaycomments for a given item on display pop-up and add new comment

### PR DESCRIPTION
**Pull request by Whoistolu**
## In this task, we implemented these:

- When the page loads, the web app uses the Involvement API to show the item likes and combines them with the data from the base API.
- When the popup loads, the web app retrieves data from the Involvement API to show the item comments.
- Implement the "add new comment" form on the Comments pop up according to the wireframe:

### CODE REVIEW

**Review by Lordkaito — CHANGES_REQUESTED** [review]
You're doing great so far! But I think there are some changes to apply:
Please kindly remove the empty files on the project, so we keep the code clean.
If you have any doubt don't hesitate to ask me.

File: src/popup.js [hunk 1]
```diff
@@ -1,37 +1,34 @@
-import { isEmpty } from "lodash";
-import { fetchAllPokemonNames } from "./index";
-
+const commentAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/yOXNdGvcjzKJL3GdYc8v/';
```
Comment (Whoistolu) [inline src/popup.js]: Kindly change the AppID so we can work on a single ID

**Review by Whoistolu — COMMENTED** [review]
Hello @Lordkaito ,

## You have done a great job so far!
However, there is a change to make.

**Review by Lordkaito — APPROVED** [review]
Good job applying the changes!
There's nothing else to say than it's time to merge it!

**Pull request merged**